pkg/destroy/powervs/cloud-transit-gateways.go

```diff
@@ -398,46 +396,14 @@ func (o *ClusterUninstaller) listTransitConnections(item cloudResource) (cloudRe
 
 // We either deal with an existing TG or destroy TGs matching a name.
 func (o *ClusterUninstaller) destroyTransitGateways() error {
-	var (
-		client *powervsconfig.Client
-		tgID   string
-		item   cloudResource
-		err    error
-
-		ctx    context.Context
-		cancel func()
-	)
-
-	ctx, cancel = contextWithTimeout()
-	defer cancel()
-
 	// Old style: delete all TGs matching by name
 	if o.TransitGatewayName == "" {
 		return o.innerDestroyTransitGateways()
 	}
 
-	// New style: delete just TG connections for existing TG
-	client, err = powervsconfig.NewClient()
-	if err != nil {
-		return err
-	}
-
-	tgID, err = client.TransitGatewayID(ctx, o.TransitGatewayName)
-	if err != nil {
-		return err
-	}
-
-	item = cloudResource{
-		key:      tgID,
-		name:     o.TransitGatewayName,
-		status:   "",
-		typeName: transitGatewayConnectionTypeName,
-		id:       tgID,
-	}
-
-	err = o.destroyTransitGatewayConnections(item)
-
-	return err
+	// New style: leave the TG and its existing connections alone
+	o.Logger.Debugf("destroyTransitGateways: tgName passed in.  Do not cleanup transit gateways.")
```

Comment:
Maybe this could be reworded a bit and become `o.Logger.Infof`? It seems to be important info for users that we're skipping the destroy on purpose. WDYT?

Reply:
That sounds reasonable.  How about this?  If not, please specify what you want.